package/network/services/ppp/patches/600-Revert-pppd-Use-openssl-for-the-DES-instead-of-the-l.patch

```diff
@@ -1,89 +0,0 @@
-From 831dca008699d485f2c8e91749657ef2d0b06166 Mon Sep 17 00:00:00 2001
-From: Martin Schiller <ms@dev.tdt.de>
-Date: Thu, 6 Dec 2018 08:43:17 +0100
-Subject: [PATCH] Revert "pppd: Use openssl for the DES instead of the libcrypt
- / glibc"
-
-For musl and glibc2.27 we can keep linking to crypt; however if we
-switch to glibc 2.28 we will have to link to one of the SSL libraries.
-
-This reverts commit 3c7b86229f7bd2600d74db14b1fe5b3896be3875.
```

Comment:
Please keep this change to not depend on openssl. We patches glibc to add DES too.

Reply:
> Please keep this change to not depend on openssl. We patches glibc to add DES too.

Hi. Could you please help me to understand then the purpose of ppp-des.c in upstream? Doesn't it do the same?

Reply:
checked a bit deeper. No changes needed. No openssl dependency present anyway. So, ppp doesn't need either openssl or  patchred glibc

Reply:
ppp should not depend on openssl by default. I am not sure how common ppp connections using DES are in the field. If ppp is able to use DES without openssl and this patch now tat would be nice.

Reply:
@hauke:
- https://github.com/ppp-project/ppp/pull/362
- https://github.com/enaess/ppp/blob/master/pppd/ppp-des.c

Reply:
@icegood just submitted an PR to fix that in 2.5.+. Apologies for that stray openssl include. Technically, you should be able to compile pppd without eaptls or peap features enabled, which allows you to use the built in sha1/md5/md4 and des/ecb functions.

Reply:
ok, this should be fine PPP now ships its own DES functionality.